### ISSUE TITLE
Form field missing a label (IU Accessibility Mandate)

### ISSUE THREAD

**Issue by mdalmau** [IUBLibTech/newton_chymistry#138]
20 occurrences in the [LSA page](https://newton.dlib.indiana.edu/newton/lsa/index.php): radio buttons need labels. For example:
`<input type="radio" value="wholedocs" name="lsa-searchradio" for="select to run LSA against the whole document">`

--------------------------
IU has a mandate that all web sites published on IU servers must meet WCAG I and II Accessibility requirements. If the requirements are NOT met, sites will be taken down. To learn more about this initiative, visit the [Digital Strategy Initiative](https://ucm.iu.edu/digital-strategy/index.html) web site.

- [x] Ask Bill who should be added to the SiteImprove tool

**Comment by mdalmau** [comment]
@wehooper : I requested SiteImprove access for you. Once you have access, you will login using your @iu credentials. If you want to read more about SIteImprove and accessibility, LOGIN to the KB and [start here](https://kb.iu.edu/d/webs). 

If you plan to make local fixes, you will need to install the SiteImprove Plugin for your preferred browser, to test your changes.

**Comment by wehooper** [comment]
The SI crawler identified 39 <input> elements without a label. Each of those elements had an external HTML <label> visible to the user. To resolve the issue, I added an @aria_label attribute to each input with its respective external label as the attribute value. That was sufficient to make the issue disappear from the list of issues found by the SI crawler after a pass. We can close this when the LSA component can be updated from github/IUB_LIBTECH/lsa.

**Comment by mdalmau** [comment]
@wehooper : could you create a pull request for this issue and the other accessibility HTML title issue? I am not sure if this needs to go on the LSA branch or on the primary (master) branch. Thanks!

**Comment by wehooper** [comment]
@mdalmau : the changes requested under this issue were successfully made to a separate webapp in a different github repository. No changes were made to our 'newton_chymistry' here, and without a change we can't define a new pull request on 'main'. Closing with comment.

**Comment by wehooper** [comment]
Oops, okay.  Above my pay grade.  : )
